app/Filters.tsx

```diff
@@ -82,7 +82,7 @@ export default function ModalScreen() {
 				onFocus={() => setIsFocusCategory(true)}
 				onBlur={() => setIsFocusCategory(false)}
 				onChange={item => {
-					setCategoryId(item.id);
+					setCategoryId(prev => item.id);
```

Comment:
```suggestion
					setCategoryId(item.id);
```
We use lambda only when we want to transform an existing state. Old form was better.

Reply:
Done